### ISSUE TITLE
Mount NFS Unavailable

### ISSUE THREAD

**Issue by sl0wz3r** [IceWhaleTech/ZimaOS#120]
Currently there is no provision to mount NFS either in the GUI or from a Terminal.

**Comment by ShiroDN** [comment]
Yes, the ability to mount NFS from a remote target is really needed. With CasaOS it was easy to do from the underlying OS (fstab), but in ZimaOS, which has a read-only filesystem and possibly regenerates its config on updates, it’s hard to do.
